boards/FSM/Src/main.c

```diff
@@ -222,6 +224,9 @@ int main(void)
         Io_MSP3002K5P3N1_GetPressurePsi, MIN_BRAKE_PRESSURE_PSI,
         MAX_BRAKE_PRESSURE_PSI);
 
+    brake_actuation_status =
+        App_BinarySwitch_Create(Io_MSP3002K5P3N1_BrakeIsActuated);
```

Comment:
I thought about this some more and think ultimately the best approach is to just create a 
```
App_Brake_Create();
App_Brake_Destroy();
App_Brake_IsActuated();
```
`BinarySwitch` or any similar abstract concept just doesn't quite cut it because `ON/OFF` doesn't quite express `IsActuated` clearly enough.

Reply:
Rescoped for the Nth time. Going with a `BinaryStatus` instead.
```
struct BinaryStatus* brake_actuation_status = 
     App_BinaryStatus_Create(Io_XXXX_IsBrakeActuated);

App_BinaryStatus_IsActive(brake_actuation_status);
```